actionmailer/lib/action_mailer/log_subscriber.rb

```diff
@@ -6,9 +6,16 @@ module ActionMailer
   class LogSubscriber < ActiveSupport::LogSubscriber
     # An email was delivered.
     def deliver(event)
-      info do
-        recipients = Array(event.payload[:to]).join(', ')
-        "\nSent mail to #{recipients} (#{event.duration.round(1)}ms)"
+      if exception = event.payload[:exception]
+        error do
+          recipients = Array(event.payload[:to]).join(', ')
```

Comment:
there're several other places where `"#{event.duration.round(1)}ms"` shows up.. how about we make a private method like:

```
def duration(event)
  "#{event.duration.round(1)}ms"
end
```

to further reduce duplication?